modules/textures/package.json

```
@@ -36,7 +36,7 @@
     "pre-build": "npm run copy-libs && npm run build-worker && npm run build-bundle && npm run build-bundle -- --env.dev",
     "copy-libs": "cp -rf ./src/libs ./dist/libs",
     "build-bundle": "webpack --display=minimal --config ../../scripts/bundle.config.js",
-    "build-worker": "webpack --entry ./src/workers/basis-loader.worker.js --output ./dist/basis-loader.worker.js --config ../../scripts/worker-webpack-config.js && webpack --entry ./src/npy-loader.worker.js --output ./dist/npy-loader.worker.js --config ../../scripts/worker-webpack-config.js"
+    "build-worker": "webpack --entry ./src/workers/basis-loader.worker.js --output ./dist/basis-loader.worker.js --config ../../scripts/worker-webpack-config.js && webpack --entry ./src/workers/npy-loader.worker.js --output ./dist/npy-loader.worker.js --config ../../scripts/worker-webpack-config.js && webpack --entry ./src/workers/compressed-texture-loader.worker.js --output ./dist/compressed-texture-loader.worker.js --config ../../scripts/worker-webpack-config.js && webpack --entry ./src/workers/crunch-loader.worker.js --output ./dist/crunch-loader.worker.js --config ../../scripts/worker-webpack-config.js"
```

Nit: Getting long, maybe split in scripts
```suggestion
    "build-workers": "build-basis-worker && build-...",
    "build-basis-worker": "webpack --entry ./src/workers/basis-loader.worker.js --output ./dist/basis-loader.worker.js --config ../../scripts/worker-webpack-config.js",
    ....
```